source/scripts/components/Snackbar/snackbar.css

```diff
@@ -0,0 +1,42 @@
+.snackbar {
+  position: fixed;
+  top: 90vh;
+  left: 50%;
+  min-width: 200px;
+  height: 50px;
```

Comment:
Long text will break fixed height layout. Maybe it's better to remove fixed height so Snakbar will grow naturally with content?